docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -0,0 +1,156 @@
+---
+id: Specification Digital Twin Kit
+title: Specification
+description: 'Digital Twin Kit'
+sidebar_position: 4
+---
+
+### Digital Twin Kit
+
+<!--
+Development View of the Kit.
+-->
+
+<!-- !Mandatory! -->
+## API Specifications
+
+All openAPI-specifications for the Digital Twin KIT services are rendered in the section [of these docs](API Specs)
+
+### Asset Administration Shell
+
+The Asset Administration Shell is a specification that is released by the IDTA with a perspective to be adopted bei the IEC. 
+Its mission is defining how “information about assets […] can be exchanged in a meaningful way between partners in a value
+creation network”. As such, it is well-suited to contribute to the toolbox of Catena-X. While the Spec offers an extensive
+suite of meta-model elements and APIs, Catena-X only uses a small subset. What exactly is defined in the Catena-X standard 
+CX - 0002.
+
+#### Submodels
+
+An Asset Administration Shell is organized in Submodels. Each Submodel represents a self-contained aspect of an asset - 
+typical examples are the Nameplate or AssemblyPartRelationship (which denotes the hierarchical composition of parts into
+a whole). As different aspect of an Asset may be known to different parties on the value-chain, submodels for a single asset
```

Comment:
As different aspects of an asset may be known to different parties in the value chain, submodels for a single asset